docs/usage/self-hosted-configuration.md

```diff
@@ -553,6 +553,11 @@ You can choose from the following behaviors for the `dryRun` config option:
 
 Information provided mainly in debug log level.
 
+## encryptedWarning
```

Comment:
Here's the wishlish from `@rarkins`, as todos:


- [ ] A new page on `docs.renovatebot.com` explaining our preference for UI-uploaded secrets, and a section on how to migrate
- [ ] A warning message printed once per repo/job, which says something like "Support for encrypted secrets in config is deprecated by the Mend Renovate App" plus a link to this documentation page
- [ ] The warning is shown on Dependency Dashboard, and an indicator is shown in each PR mentioning that warnings were raised (not sure if we need the full message or not)

Source:

- https://github.com/renovatebot/renovate/discussions/28895#discussioncomment-9353343

Reply:
1. we would like first to support migrating secrets in production before we add it to the docs.
2. the message itself will be set on our Mend App with the doc link.
3. i will probably add the PR warnings to this PR soon, i wasn't sure if we need it or not.

Reply:
i wrote a comment in the discussion asking how to do this from product perspective, resolving this comment.